ts_src/psbt.ts

```diff
@@ -687,43 +706,47 @@ export class Psbt {
     const inputAgs: Buffer[] = [];
     const inputValues: string[] = [];
 
+    // counter for fetching blinding private key
+    let confidentialUtxoIndex = 0;
+
     // unblind inputs
     this.data.inputs.forEach((input: PsbtInput, index: number) => {
-      let prevout: any;
+      let prevout: WitnessUtxo;
```

Comment:
it was `any` because of the input.nonWitnessUtxo branch. This assigns to prevout a transaction.Output type rather than WitnessUtxo. Is it safe like that?

Reply:
Yes because `WitnessUtxo` and `Output` are the same interface... but I will add `as WitnessUtxo`